src/openloco/graphics/gfx.cpp

```diff
@@ -41,9 +41,15 @@ namespace openloco::gfx
     constexpr uint32_t g1_count_temporary = 0x1000;
 
     static loco_global<drawpixelinfo_t, 0x0050B884> _screen_dpi;
```

Comment:
0x50B884 -> 0x50B8AE is the `screen_info_t` see `ui.cpp` could you modify these functions to use a reference to the screenInfo instead of all of these additional globals. I'm not 100% sure where best to locate all of these things.